zipkin-ui/js/component_ui/traceToMustache.js

```diff
@@ -87,7 +87,9 @@ export function formatEndpoint({ipv4, port = 0, serviceName = ''}) {
 }
 
 export default function traceToMustache(trace) {
+  const raw = trace;
```

Comment:
The raw data doesn't belong inside the modelview; this function is supposed to transform the incoming raw data into a format that is ready for mustache rendering.
Can you try refactoring, so that raw is not passed as part of the modelview?